packages/countries/scripts/generateCountrySymbol.mjs

```diff
@@ -0,0 +1,404 @@
+
+    const countryFileName = componentName + ".tsx";
+
+    let viewBox;
+    const newFilePath = path.join(componentsPath, componentName + ".tsx");
+
+    countryMetaMap[countryCode] = {
+      countryCode,
+      countryName,
+      countryNameSanitized,
+      componentName,
+      textName: baseCountryName,
+      countryFileName,
+    };
+
+    console.log("processing", fileName, "to", newFilePath);
+
+    // SVGO is a separate step to enable multi-pass optimizations.
+    const optimizedSvg = optimize(svgString, {
+      multipass: true,
+      plugins: [
+        {
+          name: "preset-default",
+          params: {
+            overrides: {
+              // makes country symbol scaled into width/height box
+              removeViewBox: false,
+            },
+          },
+        },
+        {
+          name: "removeAttrs",
+          params: {
+            attrs: "(width|height)",
+          },
+        },
+      ],
+    });
+
+    const svgPaths = optimize(optimizedSvg.data, {
+      plugins: [
+        {
+          name: "mapHTMLAttributesToReactProps",
+          fn: () => {
+            return {
+              element: {
+                enter: (node) => {
+                  const newAttributes = {};
+                  // preserve an order of attributes
+                  for (const [name, value] of Object.entries(node.attributes)) {
+                    if (
+                      node.name === "mask" &&
+                      name === "style" &&
+                      typeof value === "string" &&
+                      value.includes("mask-type:")
+                    ) {
+                      newAttributes["mask-type"] = value.slice(10);
+                    } else {
+                      newAttributes[svgAttributeMap[name] || name] = value;
+                    }
+                  }
+                  node.attributes = newAttributes;
+                },
+              },
+            };
+          },
+        },
+        {
+          name: "find-viewBox",
+          fn: () => {
+            return {
+              element: {
+                enter: (node, parentNode) => {
+                  if (parentNode.type === "root") {
+                    viewBox = node.attributes.viewBox;
+                  }
+                },
+              },
+            };
+          },
+        },
+        {
+          name: "removeSvg",
+          fn: () => {
+            return {
+              element: {
+                exit: (node, parentNode) => {
+                  if (node.name === "svg") {
+                    const index = parentNode.children.indexOf(node);
+                    parentNode.children.splice(index, 1, ...node.children);
+                  }
+                },
+              },
+            };
+          },
+        },
+      ],
+    });
+
+    const fileContents = Mustache.render(template, {
+      svgElements: svgPaths.data,
+      componentName,
+      ariaLabel: baseCountryName.toLowerCase(),
```

Comment:
could this not change the reader's emphasis?
```suggestion
      ariaLabel: baseCountryName,
```

Reply:
Yeah potentially, I based it on what we were doing with the original icon script so thought there might be a reason